### DISCUSSION TITLE
Download links for RustDesk Clients

### DISCUSSION THREAD

**Discussion by GuillaumeHullin** [rustdesk/rustdesk#9515] [General]
I would like to make a download link which would point automatically to the latest version.

Following this https://docs.github.com/en/repositories/releasing-projects-on-github/linking-to-releases
I could make up this https://github.com/rustdesk/rustdesk/releases/latest/download/rustdesk-1.3.1-x86_64.exe 
But still including the version number...

What I'm trying to solve is to have clients which low-knowledge of computer easily downloading RustDesk... I guess that's what scammer wants too.

One other way would be to have a page on rustdesk.com with a nice UI (and a big warning message), with simple download link without going via GitHub release page.

**Comment by TechZeug** [comment]
Have you found a solution? I would like to provide a link for my customers on my website and always load the latest version.

**Comment by xlionjuan** [comment]
https://github.com/rustdesk/rustdesk/releases/latest

**Comment by GuillaumeHullin** [comment]
@TechZeug: unfortunately I haven't seen any yet.
@xlionjuan: not exactly what's asked... A GitHub release page is not scary for us but for the rest of common mortals... the difference between 32bit Windows and 64bit is uncertain at best... and then choosing MSI or EXE is just a lucky guess...

**Reply by xlionjuan** [reply]
So you should tell them which to choose.....

**Reply by GuillaumeHullin** [reply]
Sure @xlionjuan, but as a tech support for people sometimes on the older generation... and non-english speaking... it is un-necessary challenging.
I do see the point of scaring people from downloading Rustdesk when they have a scammer on the phone... but still it could be a good idea to not make it too hard for legit use. Hence my original idea which is to have download link on the main website with a UI to warn people away if they are scammed and maybe some delaying popup.

**Reply by xlionjuan** [reply]
Or you copy and paste the link for them, or own your shortlink URL and replace it when new version comes.

**Comment by abdullah-erturk** [comment]
> I would like to make a download link which would point automatically to the latest version.
> 
> Following this https://docs.github.com/en/repositories/releasing-projects-on-github/linking-to-releases I could make up this https://github.com/rustdesk/rustdesk/releases/latest/download/rustdesk-1.3.1-x86_64.exe But still including the version number...
> 
> What I'm trying to solve is to have clients which low-knowledge of computer easily downloading RustDesk... I guess that's what scammer wants too.
> 
> One other way would be to have a page on rustdesk.com with a nice UI (and a big warning message), with simple download link without going via GitHub release page.

Always download the release latest version and if you have your own server you can point it to your own server as soon as rustdesk is installed.

https://github.com/abdullah-erturk/RustDesk-Silent-Installation
